FrontPage/schema/PostTaskViewController.graphql

```diff
@@ -0,0 +1,39 @@
+mutation deleteTask ($id: ID!){
```

Comment:
Is there a better name for this file?

Reply:
any suggestions? I think "deleteTask" is very well explaining what it does.